src/coreclr/jit/fgdiagnostic.cpp

```diff
@@ -3534,12 +3534,22 @@ void Compiler::fgDebugCheckNodesUniqueness()
 //    - If the method has natural loops, the loop table is not null
 //    - Loop `top` must come before `bottom`.
 //    - Loop `entry` must be between `top` and `bottom`.
+//    - Children loops of a loop are disjoint.
 //    - All basic blocks with loop numbers set have a corresponding loop in the table
 //    - All basic blocks without a loop number are not in a loop
 //    - All parents of the loop with the block contain that block
+//    - If the loop has a pre-header, it is valid
+//    - The loop flags are valid
 //
 void Compiler::fgDebugCheckLoopTable()
 {
+#ifdef DEBUG
+    if (verbose)
+    {
+        printf("*************** In fgDebugCheckLoopTable\n");
+    }
+#endif // DEBUG
```

Comment:
Can't these 6 lines be replaced by a single `JITDUMP`? (which takes care of verbose and DEBUG under the hood)

Reply:
Yes, but there is a pattern for the "In ..." prints like this that is often used so you can set a breakpoint on the "print" line that will only hit if COMPlus_JitDump is set to the function of interest. Can't do that with JITDUMP without conditional breakpoints. A minor thing, but sometimes convenient.